puppet/modules/jenkins_job_builder/files/config/yaml/jobs/packaging_build_deb_coreproject.yaml

```diff
@@ -20,8 +20,8 @@
           name: os
           values:
           - wheezy
+          - jessie
```

Comment:
Could you either revert this part until we have the sources in foreman-packaging, or change the combination filter to not build jessie by default?  (replace squeeze with jessie)